WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeButtonPrefsFragment.java

```diff
@@ -116,6 +116,11 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         mPrefAllowCommentLikes = (WPPrefView) view.findViewById(R.id.pref_allow_comment_likes);
         mPrefTwitterName = (WPPrefView) view.findViewById(R.id.pref_twitter_name);
 
+        if (!mSite.isWPCom() && mSite.isJetpackConnected()) {
```

Comment:
Just a quick note: we prefer to use `mSite.isUsingWpComRestApi()` instead of `mSite.isJetpackConnected()` because a site could be Jetpack connected but still using the xml-rpc api. For this feature it doesn't make a difference since we don't show the "Sharing" button to xml-rpc users. Here is a test which shows this case: https://github.com/wordpress-mobile/WordPress-FluxC-Android/blob/develop/example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_SiteTestJetpack.java#L146